src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -140,8 +148,19 @@ public void initialize() {
       selectionModel.select(newMapIndex);
       mapListView.scrollTo(newMapIndex);
     });
-
-    featuredModListView.setCellFactory(param -> new StringListCell<>(FeaturedMod::getDisplayName));
+    
+    Function<FeaturedMod, String> isDefaultModString = mod ->
+      Objects.equals(mod.getTechnicalName(), KnownFeaturedMod.DEFAULT.getTechnicalName()) ?
+      " " + i18n.get("game.create.defaultGameTypeMarker") : null;
+    
+    featuredModListView.setCellFactory(param ->
+        new DualStringListCell<FeaturedMod>(FeaturedMod::getDisplayName, isDefaultModString, false) {
+      @Override
+      protected void init() {
+        // use bind to overwrite updates from css
+        right.textFillProperty().bind(new SimpleObjectProperty<Paint>(Color.GREY));
```

Comment:
Try avoid UI style set in java code at all costs, use Pseudoclass

Reply:
I can try but it might not be possible due to how `DualStringListCell` works.

Reply:
it is just give it a style class(possibly in code) and set this style class in css

Reply:
Ok problem was that (default) ended up with other font/size than the other cell. Need to check when i have time.